apps/scan/frontend/src/components/display_settings_button.tsx

```diff
@@ -21,10 +21,13 @@ export function DisplaySettingsButton(): JSX.Element | null {
   }
 
   return (
-    <Button onPress={history.push} value={Paths.DISPLAY_SETTINGS}>
```

Comment:
fixing broken `onPress` handler here - the `Button` was recently changed to fire `onPress` with two arguments instead of one